packages/core/services/FileStorageServiceBase/index.ts

```diff
@@ -0,0 +1,36 @@
+import axios from "axios";
+
+import HttpServiceBase from "../HttpServiceBase";
+
+export default class FileStorageServiceBase extends HttpServiceBase {
```

Comment:
Changing this to an abstract class would let you move the methods defined in `FileDownloadService` (the interface) thereby letting you not need both a `FileDownloadService` interface + `FileDownloadServiceBase` class but instead just one `FileDownloadService` abstract class.
```suggestion
export default abstract class FileStorageService extends HttpServiceBase {
```

Reply:
resolved in [10d7d67](https://github.com/AllenInstitute/biofile-finder/pull/204/commits/10d7d67753c85570a72d61de8bc8a191b4be552d)